core/ffmpeg/ffmpeg.go

```diff
@@ -27,12 +29,19 @@ func New() FFmpeg {
 }
 
 const (
-	// OGG cover uses "Theora", which is not supported by the browser's image tag.
-	// However, we can tell ffmpeg to perform the image conversion for us.
-	extractImageCmd = "ffmpeg -i %s -an -c:v mjpeg -f image2pipe -"
+	extractImageCmd = "ffmpeg -i %s -an -c:v %i -f image2pipe -"
 	probeCmd        = "ffmpeg %s -f ffmetadata"
 )
 
+var (
+	//   Stream #0:0: Video: theora, yuv444p, 214x152 [SAR 1:1 DAR 107:76], 25 tbr, 25 tbn, 25 tbc
+	//   Stream #0:0: Video: webp, yuv420p
+	//   Stream #0:0: Video: mjpeg (Baseline),
+	//   Stream #0:0: Video: png, rgb24(pc),
+	videoStreamTypeRegex   = regexp.MustCompile(`Stream #\d+:\d*(?:\(\w+\))?: Video: ([^\s,]+)`)
+	supportedRawStreamType = []string{"webp", "mjpeg", "png"}
+)
```

Comment:
gif should be supported on all clients (Web, Android, ios) natively. Maybe also avif, not sure. 

Consider putting this hardcoded list into an environment variable, so its changeable later on?

Reply:
Not sure about having this as a config - it feels like unneeded level of complexity to preserve the raw image stream that may or may not be supported in users' device.

To be honest I'm not even sure if `webp` was a good choice considering external media player support...

Reply:
Not all clients support Gif, at least not without knowing ahead of time that the image is a Gif (because they may have to use a different codepath to support it), and the Subsonic API unfortunately doesn't have a format attribute on the image assets.